lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -449,7 +449,7 @@ export default class BitcoinProcessor {
       for (const transaction of transactions) {
         const block = blockMetaDataMap.get(transaction.transactionTime);
         if (block !== undefined) {
-          transaction.normalizedTransactionFee = block.normalizedFee;
+          transaction.normalizedTransactionFee = Math.floor(block.normalizedFee);
```

Comment:
Why do we floor here instead of in the fee calculator?

Reply:
Discussed offline.